### PR TITLE
Jenkinsfile: Support specifying tags as VCS revisions

### PR DESCRIPTION
**Pull request by sschuberth**
Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch-si.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -39,6 +39,12 @@ pipeline {
             description: 'Allow dynamic versions of dependencies (support projects without lock files)'
         )
 
+        booleanParam(
+            name: 'USE_CLEARLY_DEFINED_CURATIONS',
+            defaultValue: true,
+            description: 'Use package curation data from the <a href="https://clearlydefined.io/">ClearlyDefine</a> service'
```
Comment (fviernau) [inline Jenkinsfile]: ClearleyDefine**d**: missing a `d`

Comment (fviernau) [inline Jenkinsfile]: ClearleyDefine**d**: missing a `d`

Comment (sschuberth) [reply Jenkinsfile]: Done. FYI, I had to remove the link as it does not render as a link in Jenkins by default.

Comment (fviernau) [reply Jenkinsfile]: Thanks!

**Pull request merged**